public-key

```diff
@@ -0,0 +1,41 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Please remove this file.

Reply:
Oops!

Reply:
I thought I had removed this weeks ago.  It is gone now.